src/main/java/com/google/api/codegen/py/PythonSphinxCommentFixer.java

```diff
@@ -22,6 +22,8 @@
 
   /** Returns a Sphinx-formatted comment string. */
   public static String sphinxify(String comment) {
+    // escape '$' in the comment first
+    comment = comment.replaceAll("\\$", "\\\\\\$");
```

Comment:
👍 to @jmuk's suggestion. We're not doing any abstract pattern matching here. Literals are fine. (i.e. `fgrep` rather than `egrep`)